snuba/datasets/storages/transactions_common.py

```diff
@@ -101,6 +101,7 @@
         ("transaction_source", String()),
         ("timestamp", DateTime(Modifiers(readonly=True))),
         ("group_ids", Array(UInt(64))),
+        ("app_start_type", String(Modifiers(readonly=True))),
```

Comment:
Why do you add `readonly=True` modifier to this column? Is this column going to be materialized? If not, I would recommend that you remove `readonly=True` modifier

Reply:
What exactly do you mean by `materialized`? I am OK with removing it.